festim/concentration/traps/traps.py

```diff
@@ -120,7 +120,12 @@ def solve_extrinsic_traps(self):
                 du_t = f.TrialFunction(trap.density[0].function_space())
                 J_t = f.derivative(trap.form_density, trap.density[0], du_t)
                 problem = festim.Problem(J_t, trap.form_density, [])
+
+                f.begin(
+                    "Solving nonlinear variational problem."
+                )  # Add message to fenics logs
                 trap.newton_solver.solve(problem, trap.density[0].vector())
+                f.end()
```

Comment:
So, I added `fenics.begin()`/`fenics.end()` (see [decription](https://fenicsproject.org/olddocs/dolfin/1.4.0/python/programmers-reference/cpp/common/begin.html)) to increase the indentation level of info messages printed by the Newton solver.